File: solutions/uri/1005/1005.c
#include <stdio.h>

int main() {
    float a, b;

    scanf("%f", &a);
    scanf("%f", &b);

    printf("MEDIA = %.5f\n", (a * 3.5 + b * 7.5) / 11.0);

    return 0;
}

File: solutions/uri/1005/1005.go
package main

import "fmt"

func main() {
	var a, b float64

	fmt.Scanf("%f", &a)
	fmt.Scanf("%f", &b)

	fmt.Printf("MEDIA = %.5f\n", (a*3.5+b*7.5)/11.0)
}
